db/migrate/20240219160343_add_vat_chargeable_to_lead_providers.rb

```diff
@@ -0,0 +1,5 @@
+class AddVatChargeableToLeadProviders < ActiveRecord::Migration[7.1]
+  def change
+    add_column :lead_providers, :vat_chargeable, :boolean, default: true
```

Comment:
are we sure we use `vat_chargeable` in NPQ calculations, I couldn't find it in the ECF app, do you mind pointing me to it?

Reply:
I had assumed since its in the schema for NPQ lead providers that we would want it, but it doesn't look like its used at the moment (just in the ECF statement calculator). I'll remove it if its not going to be needed.